deploy/kubedirector/kubedirector.hpe.com_kubedirectorapps_crd.yaml

```diff
@@ -4,198 +4,386 @@ metadata:
-                        pattern: '^([0-9.]+)([eEinumkKMGTP]*[-+]?[0-9]*)$'
-            config:
-              type: object
-              required: [selectedRoles, roleServices]
-              properties:
-                configMeta:
-                  type: object
-                  nullable: true
-                selectedRoles:
-                  type: array
-                  items:
-                    type: string
-                    minLength: 1
-                roleServices:
-                  type: array
-                  items:
-                    type: object
-                    required: [roleID, serviceIDs]
-                    properties:
-                      roleID:
-                        type: string
-                        minLength: 1
-                        maxLength: 63
-                        pattern: '^(([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9])?$'
-                      serviceIDs:
-                        type: array
-                        items:
+                      type: object
+                      required: [roleID, serviceIDs]
+                      properties:
+                        roleID:
                           type: string
                           minLength: 1
-                          maxLength: 15
-                          pattern: '^[a-z0-9]([-a-z0-9]*[a-z0-9])?$'
-            defaultPersistDirs:
-              type: array
-              items:
-                type: string
-                pattern: '^/.*[^/]$'
-            defaultEventList:
-              type: array
-              items:
-                type: string
-                pattern: '^configure$|^addnodes$|^delnodes$'
-            capabilities:
-              type: array
-              items:
-                type: string
-                minLength: 1
-            systemdRequired:
-              type: boolean
\ No newline at end of file
+                          maxLength: 63
+                          pattern: '^(([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9])?$'
+                        serviceIDs:
+                          type: array
+                          items:
+                            type: string
+                            minLength: 1
+                            maxLength: 15
+                            pattern: '^[a-z0-9]([-a-z0-9]*[a-z0-9])?$'
+              defaultPersistDirs:
+                type: array
+                items:
+                  type: string
+                  pattern: '^/.*[^/]$'
+              defaultEventList:
+                type: array
+                items:
+                  type: string
+                  pattern: '^configure$|^addnodes$|^delnodes$'
+              capabilities:
+                type: array
+                items:
+                  type: string
+                  minLength: 1
+              systemdRequired:
+                type: boolean
+  conversion:
+    strategy: Webhook
+    conversionReviewVersions: ["v1beta1", "v1beta2"]
```

Comment:
This should only have v1beta1 here. This is talking about the version of the ConversionReview object I think, not the version of the CR.

Reply:
I am just getting rid of the conversionReviewVersions field as it is not necessarily needed by v1beta1 apiextension version.